apps/mux/frontend/package.json

```diff
@@ -19,14 +18,14 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@contentful/app-scripts": "^1.29.0",
     "@contentful/app-sdk": "4.12.0",
     "@contentful/dam-app-base": "2.0.46",
     "@contentful/f36-components": "4.67.0",
     "@contentful/react-apps-toolkit": "1.2.16",
     "@mux/mux-player-react": "2.9.1",
     "@mux/mux-uploader-react": "1.0.1",
     "@vitejs/plugin-react": "^4.3.2",
-    "crypto-browserify": "^3.12.0",
```

Comment:
This library had a low severity vulnerability and I noticed the package isn't even being used, so that's why I just removed it.

Reply:
good catch